libs/ui-toolkit/datepicker/src/lib/adapter/datepicker.adapter.ts

```diff
@@ -23,10 +25,22 @@ export class GorilaDatepickerAdapter extends MomentDateAdapter {
     return super.parse(value, this.getDateFormat() || parseFormat);
   }
 
-  public format(date: Moment, displayFormat: string): string {
+  public format(date: DateRange<Moment> | Moment, displayFormat: string): string {
+    if (date instanceof DateRange) {
+      return super.format(date.start, this.getDateFormat() || displayFormat);
+    }
     return super.format(date, this.getDateFormat() || displayFormat);
   }
 
+  public isValid(date: DateRange<Moment> | Moment): boolean {
+    if (date instanceof DateRange) {
+      const startIsValid = date.start && this.clone(date.start).isValid();
+      const endIsValid = date.end === null || this.clone(date.end).isValid();
```

Comment:
Não tenho certeza se o caso `date.end === undefined` é possível e se `clone` tem alguma proteção contra ele, mas eu colocaria para garantir: `date.end === null || (date.end && this.clone(date.end).isValid())`